### DISCUSSION TITLE
Safety info?

### DISCUSSION THREAD

**Discussion by Djspac3** [geode-sdk/sdk#554] [Help]
What is the info for what is safe, or what can this software do like dangerously or what?

Even what is the tos?

(Request if there aren't any, please, when you add them, add a user-friendly version too)
(And if there is just, please have a user-friendly version)

**Comment by Djspac3** [comment]
not using discord bc i dont have an account and i won't make one

**Reply by heavyutils** [reply]
Idk abt the question
